PDP10/ka10_dpy.c

```diff
@@ -180,10 +180,24 @@ const char *dpy_description (DEVICE *dptr)
 /* until it's done just one place! */
 static void dpy_set_int_done(UNIT *uptr)
 {
-    uptr->STAT_REG |= CONI_INT_DONE;
+    uptr->DELAYED_DONE = 1;
     uptr->INT_COUNTDOWN = INT_COUNT;
 }
 
+/* update interrupt request */
+static void check_interrupt (UNIT *uptr)
+{
+    if (uptr->STAT_REG & CONI_INT_SPEC) {
+        uint32 sc = uptr->STAT_REG & CONX_SC;
+        set_interrupt(DPY_DEVNUM, sc >> CONX_SC_SHIFT);

```

Comment:
Is it OK to call set_interrupt with a zero value? Same thing at line 195.  Would be nice if the comment at 187 explains who calls this.  The name "check_interrupt" sounds a little like this is about interrupt service, when it's about sending interrupts.

Reply:
Good points.  Yes, that should be ok.  The set_interrupt function checks for zero itself.